src/style.css

```diff
@@ -1,3 +1,70 @@
+:root {
+  --main-color-font: #000;
+  --main-color-font-hover: #00acdc;
+}
+
+* {
```

Comment:
It's good to also add *::after and *::before selectors in this case.

Reply:
Fixed. New changes pushed.